cylc/flow/scheduler.py

```diff
@@ -172,7 +172,6 @@ class Scheduler:
     contact_data: Optional[dict] = None
 
     # run options
-    is_restart: bool = False
```

Comment:
The idea of specifying everything up here is so that we can see all of the Scheduler attributes in one place. Can always default to `None` until set later.